/**:
  ros__parameters:
    vehicle:
      wheel_radius: 0.37 # Measured from SVL
      wheel_width: 0.27 # Measured from SVL
      wheel_base: 2.734 # Measured from SVL
      wheel_tread: 1.571 # Measured from SVL
      overhang_front: 1.033 # Measured from SVL
      overhang_rear: 1.021 # Measured from SVL
      overhang_left: 0.3135 # Measured from SVL
      overhang_right: 0.3135 # Measured from SVL
      vehicle_height: 1.662 # Measured from SVL
      cg_to_rear: 1.367 # Assuming it is in middle of front and rear axle centers
      tire_cornering_stiffness_front: 0.1 # Taken from AVP demo params, can't verify
      tire_cornering_stiffness_rear: 0.1 # Taken from AVP demo params, can't verify
      mass_vehicle: 2120.0 # Measured from SVL
      inertia_yaw_kg_m2: 12.0  # Taken from AVP demo params, can't verify
      maximum_turning_angle_rad: 0.53


